<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePlatform</class>
 <widget class="QWidget" name="CreatePlatform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create platform</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="platNameLabel">
        <property name="text">
         <string>Platform name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="platNameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="platTypeLabel">
        <property name="text">
         <string>Platform type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="platTypeComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fileComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
